benchmarks/overrides_benchmark/bench.py

```diff
@@ -0,0 +1,67 @@
+import torch
+import time
+import argparse
+
+from common import SubTensor, WithTorchFunction, SubWithTorchFunction
+
+NUM_REPEATS = 1000
+NUM_REPEAT_OF_REPEATS = 1000
+
+
+def bench(t1, t2):
+    bench_times = []
+    for _ in range(NUM_REPEAT_OF_REPEATS):
+        time_start = time.time()
+        for _ in range(NUM_REPEATS):
+            torch.add(t1, t2)
+        bench_times.append(time.time() - time_start)
+
+    bench_time = float(torch.min(torch.Tensor(bench_times))) / 1000
+    bench_std = float(torch.std(torch.Tensor(bench_times))) / 1000
```

Comment:
@ezyang Only division happens here, and it's by 1000, not 0. Elsewhere I'm only doing `torch.add`.

Reply:
Got it, `torch.std` has UB with a length-1 Tensor.